### DISCUSSION TITLE
Error with Sabre CalDav integration

### DISCUSSION THREAD

**Discussion by bfeddes** [YetiForceCompany/YetiForceCRM#17076] [How to]
So when I create an event in Outlook, after CRON runs that event will be added automatically to Yeti as well. However, when I create an event in Yeti that I want to be synced to Outlook, I get an error. I found the logs of this issue which can be found below.
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/112420814/6cf32f1d-144b-4adf-a99b-b1e68ac0e933)

It would appear to be some authentication issue, but I cannot be sure, I am not too sure exactly what I am looking at.

Here is my current configuration for everything:

Yetiforce Key setup:
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/112420814/d83684f7-4214-492a-9166-35a303c8753f)

My configuration using CalDav Synchronizer on Outlook works and says the connection test was successful, as shown below:
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/112420814/e8707fc7-2ace-4695-a0bf-7b63c8acf91a)

However, when I try to integrate I get this error: 
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/112420814/462e94ea-503f-487a-a50c-019c609d89a1)

Some steps I have already taken as instructed by the other discussions I read:
1. change from false to "Private" in the /Components/Dav.php folder as shown below:
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/112420814/500ae9bb-6619-47fd-8dc3-6d5a4b5737ab)

I have been working at this for hours with no resolution. If you need any additional information please be specific with what you want and I will add a reply with information.

**Comment by mariuszkrzaczkowski** [comment]
duplicate https://github.com/YetiForceCompany/YetiForceCRM/discussions/17072#discussioncomment-5857181
